src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $white;
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+  max-width: 200px;
+  &__image {
+    width: 100%;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: cover;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary;
```

Comment:
Ensure that the SCSS variable `$secondary` is defined in your `variables.scss` file or imported correctly. This is crucial for maintaining styling consistency.